tests/bin/SKIP

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
The ALL_CAPS programs are usually spread imports and I'm somewhat inclined to have it differ OR become a proper spread import (even if that takes longer). I for one would welcome a change that lets me know N tests were skipped and printing the reason for doing that.